migrations/0008_add_server-settings.sql

```diff
@@ -0,0 +1,11 @@
+-- Migration number: 0003   2023-02-24T15:03:27.478Z
+
+CREATE TABLE IF NOT EXISTS server_settings (
+  setting_name TEXT UNIQUE NOT NULL,
+  setting_value TEXT
```

Comment:
the value should probably be `NOT NULL` too? I think if a setting is null we shouldn't insert it

Reply:
good point! 👍